### ISSUE TITLE
update biskitbase docker image to include Hex 8.0

### ISSUE THREAD

**Issue by graik** [graik/biskit#27]
A new image needs to be created and uploaded to dockerstore.

- [x] clone biskitbase locally
- [x] download xplor-nih, delphi
- [x] download hex for linux to downloads
- [x] add hex installation to docker script
- [x] rebuild and test dockerbase image
- [x] upload new image to dockerstore

**Comment by graik** [comment]
currently held up by the need to upgrade delphi wrapper to version Delphi C++ version.

**Comment by graik** [comment]
Image uploaded. Delphi wrappers need to be adapted but that's another issue (see #35 and #36)
